modules/yarn.lock

```diff
@@ -13704,7 +13704,7 @@ send@0.17.1:
     range-parser "~1.2.1"
     statuses "~1.5.0"
 
-senna@2.7.9, senna@^2.3.1:
+senna@^2.3.1:
```

Comment:
2.3.1, eh. Nice.

Reply:
Huh... we almost managed to get rid of `senna` but still bringing one over through `metal-router` in `frontend-js-metal-web`... so close... :D